packages/connect/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hoprnet/hopr-connect",
-  "version": "1.93.0-next.18",
```

Comment:
These version changes are strange. Should not happen in PRs to master.

Reply:
Indeed, this appeared after rebase.